test/test_image.py

```diff
@@ -258,6 +264,41 @@ def test_write_file_non_ascii():
         assert content == saved_content
 
 
+@pytest.mark.skipif(cv2 is None, reason='opencv unavailable')
+@pytest.mark.parametrize('shape', [
+    (27, 27),
+    (60, 60),
+    (105, 105),
+])
+def test_read_1_bit_png(shape):
+    with get_tmp_dir() as root:
+        image_path = os.path.join(root, 'test_{shape}.png')
+        img = Image.new('1', shape)
+        img.save(image_path)
+        img1 = read_image(image_path)
+        img2 = cv2.imread(image_path)
+        assert img1.allclose(torch.from_numpy(img2[:, :, 0])), "Tensor mismatch!"
```

Comment:
let's use `        torch.testing.assert_close(img1[0], torch.from_numpy(img2[:, :, 0]), check_stride=False)` instead

Reply:
Is there any benefit of doing this?

Reply:
Never mind, I don't need allclose() here, I can use assert_equal() for consistency with other functions in that file, which a wrapper for torch.testing.assert_close().